docker-compose.yml

```diff
@@ -3,84 +3,97 @@ version: '2'
 services:
 
   gateway-config:
-    image: "nebraltd/hm-config:f722f1243db45f29d4d589184d61880b98938a82"
+    image: nebraltd/hm-config:711fbbd4b3c694d57263b63db52f2ebb580b40a4
+    depends_on:
+      - dbus-session
     environment:
-      - 'FIRMWARE_VERSION=2021.10.07.0'
-      - 'DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket'
-      - 'DBUS_SESSION_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket'
+      - FIRMWARE_VERSION=2021.10.07.1
+      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket
+      - DBUS_SESSION_BUS_ADDRESS=unix:path=/session/dbus/session_bus_socket
     privileged: true
-    network_mode: "host"
+    network_mode: host
     cap_add:
       - NET_ADMIN
     volumes:
-      - 'miner-storage:/var/data'
+      - miner-storage:/var/data
+      - dbus:/session/dbus
     labels:
-      io.balena.features.dbus: '1'
       io.balena.features.sysfs: '1'
       io.balena.features.kernel-modules: '1'
+      io.balena.features.dbus: '1'
```

Comment:
This commits removes quotes from most places. Do you know if the single quotes around `1` are needed?

Reply:
Yes was trying to unify the use of quotes so it matches everywhere and only where they are actually needed - predominantly on ports (due to [this](https://docs.docker.com/compose/compose-file/compose-file-v3/#ports)) 

> When mapping ports in the HOST:CONTAINER format, you may experience erroneous results when using a container port lower than 60, because YAML parses numbers in the format xx:yy as a base-60 value. For this reason, we recommend always explicitly specifying your port mappings as strings.

I didn't test the balena labels without quotes @marvinmarnold I was just going by the documentation [here](https://www.balena.io/docs/reference/supervisor/docker-compose/#labels).

Reply:
@marvinmarnold looks like they are not needed:

https://github.com/balena-io-examples/balena-netdata/blob/f485be206c1c365725b8b3dc1c0093834b20f0f8/docker-compose.yml#L18-L21

Reply:
It looks like the quotes probably would not be needed on `expose` either but perhaps to avoid confusion between this and `ports` having different ways it's best to leave both with quotes?